### ISSUE TITLE
Training errors

### ISSUE THREAD

**Issue by zhengshangdong** [yf19970118/OPLD-Pytorch#3]
Hello sir!
        Thank you for your great job! 
        I get the right results in the test_net.py. However, there are some errors when I run your train code.
        The loss is always Nan when I resume from your model_latest.pth or the original caffe weights resnet101_caffe.pth.
        Do you have any suggestions?
        Yours!

**Comment by yf19970118** [comment]
I haven't met the situation that RPN related loss becomes Nan. When generating data, we should pay extra attention to the situation that the target box is truncated into five points. I made changes in dota_devkit. This repo is separated from my other projects. I have conducted test, but I haven't retrained it. I can't rule out the possibility of bugs. I'm very sorry that I don't have time to investigate the problem in detail.

**Comment by yf19970118** [comment]
> Hello sir!
> Thank you for your great job!
> I get the right results in the test_net.py. However, there are some errors when I run your train code.
> The loss is always Nan when I resume from your model_latest.pth or the original caffe weights resnet101_caffe.pth.
> Do you have any suggestions?
> Yours!

I haven't met the situation that RPN related loss becomes Nan. When generating data, we should pay extra attention to the situation that the target box is truncated into five points. I made changes in dota_devkit. This repo is separated from my other projects. I have conducted test, but I haven't retrained it. I can't rule out the possibility of bugs. I'm very sorry that I don't have time to investigate the problem in detail.
